src/language.rs

```diff
@@ -193,5 +194,36 @@ impl From<&Language> for Vec<tree_sitter::Language> {
             Language::Toml => vec![tree_sitter_toml::language()],
             Language::TreeSitterQuery => vec![tree_sitter_query::language()],
         }
+        .into_iter()
+        .map(Into::into)
+        .collect()
+    }
+}
+
+#[cfg(target_arch = "wasm32")]
+impl From<&Language> for Vec<tree_sitter_facade::Language> {
+    fn from(language: &Language) -> Self {
+        match language {
+            Language::Bash => vec!["bash"],
+            Language::Json => vec!["json"],
+            Language::Nickel => vec!["nickel"],
+            Language::Ocaml => vec!["ocaml", "ocaml-interface"],
+            Language::OcamlImplementation => vec!["ocaml"],
+            Language::OcamlInterface => vec!["ocaml-interface"],
+            Language::Rust => vec!["rust"],
+            Language::Toml => vec!["toml"],
+            Language::TreeSitterQuery => vec!["query"],
+        }
+        .iter()
+        .map(|name| format!("tree-sitter-{}.wasm", name))
```

Comment:
These are presumably the grammar dependencies, compiled to WASM and distributed in the package (i.e., we can rely on this standard filename format)?

Also, what about `.mli`? IIRC, `Language::OCamlInterface` still uses `tree-sitter-ocaml`, but it exposes a separate `tree_sitter::Language`.

**tl;dr** If this works, then it may be worth documenting in comments why it works.